docs/bokeh/source/docs/reference/command/subcommands/settings.rst

```diff
@@ -0,0 +1,7 @@
+.. _bokeh.command.subcommands.settings:
+
+settings
+~~~~~
```

Comment:
Sphinx is very finnicky, the section underline lengths need to match the length of the section title

```suggestion
settings
~~~~~~~
```